### ISSUE TITLE
about class weights

### ISSUE THREAD

**Issue by clockwiser** [mitmul/chainer-segnet#2]
In the source, class weights are used as below.

def get_model(
        model_file, model_name, loss_file, loss_name, class_weight, n_encdec,
        n_classes, in_channel, n_mid, train_depth=None, result_dir=None):
    model = imp.load_source(model_name, model_file)
    model = getattr(model, model_name)
    loss = imp.load_source(loss_name, loss_file)
    loss = getattr(loss, loss_name)

    # Initialize
    model = model(n_encdec, n_classes, in_channel, n_mid)
    if train_depth:
model = loss(model, class_weight, train_depth)

Strangely, there is no example of setting loss and loss_name.
Without proper parameters, class weights should not work.
